bundle_pt.properties

```
@@ -217,29 +217,29 @@ are_you_sure_you_wish_to_leave_the_party_question=Tem certeza de que deseja sair
 invite=Convidar
 inviting_dots=Convidando...
 leave=Sair
-you_arent_invited_to_that_party=Você não foi convidade para esse grupo :(
+you_arent_invited_to_that_party=Você não foi convidado para esse grupo :(
```

There's no reason to have genre in this phrase

```suggestion
you_arent_invited_to_that_party=Você não recebeu um convite para este grupo :(
```